client/src/components/LibraryFolder/TopToolbar/FolderTopBar.vue

```diff
@@ -0,0 +1,382 @@
+                        <button @click="showDetails" class="primary-button toolbtn-show-locinfo" type="button">
+                            <font-awesome-icon icon="info-circle" />
+                            Details
+                        </button>
+                    </span>
+                    <div class="form-check logged-dataset-manipulation mr-1" v-if="logged_dataset_manipulation">
+                        <b-form-checkbox
+                            id="checkbox-1"
+                            :checked="include_deleted"
+                            v-on:input="toggle_include_deleted($event)"
+                            name="checkbox-1"
+                        >
+                            include deleted
+                        </b-form-checkbox>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </div>
+</template>
+<script>
+import BootstrapVue from "bootstrap-vue";
+import { getGalaxyInstance } from "app";
+import Vue from "vue";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { showLocInfo } from "./details-modal";
+import { deleteSelectedItems } from "./delete-selected";
+import { initTopBarIcons } from "components/LibraryFolder/icons";
+import mod_path_bar from "components/LibraryFolder/path-bar";
+import mod_import_dataset from "./import-to-history/import-dataset";
+import mod_import_collection from "./import-to-history/import-collection";
+import mod_add_datasets from "./add-datasets";
+import { Toast } from "ui/toast";
+import download from "./download";
+import mod_utils from "utils/utils";
+import { getAppRoot } from "onload/loadConfig";
+
+initTopBarIcons();
+
+Vue.use(BootstrapVue);
+
+export default {
+    name: "FolderTopBar",
+    props: {
+        folder_id: {
+            type: String,
+            required: true,
+        },
+        include_deleted: {
+            type: Boolean,
+            required: true,
+        },
+        selected: {
+            type: Array,
+            required: true,
+        },
+        metadata: {
+            type: Object,
+            required: true,
+        },
+        folderContents: {
+            type: Array,
+            required: true,
+        },
+    },
+    components: {
+        FontAwesomeIcon,
+    },
+    data() {
+        return {
+            dataset_manipulation: false,
+            logged_dataset_manipulation: false,
+            is_admin: false,
+            multiple_add_dataset_options: false,
+            user_library_import_dir: false,
+            library_import_dir: false,
+            allow_library_path_paste: false,
+            list_genomes: [],
+            list_extensions: [],
+            // datatype placeholder for extension auto-detection
+            auto: {
+                id: "auto",
+                text: "Auto-detect",
+                description:
+                    "This system will try to detect the file type automatically." +
```

Comment:
Recommending to use es6 multline string here instead of concatenation.

Reply:
yeah, this is inherited from legacy code. I will change :+1: